packages/battery_drivers/battery.py

```diff
@@ -98,13 +99,31 @@ def shutdown(self):
 
     def turn_off(self, timeout: int = 20, wait: bool = False, callback: Optional[Callable] = None):
         #   This is a battery shutdown, the power will be cut off after `timeout` seconds
-        timeout_str = f'{timeout}'.zfill(2)
-        self._interaction = BatteryInteraction(
-            name="turn_off",
-            command=f'Q{timeout_str}'.encode('utf-8'),
-            check=lambda d: d.get('TTL(sec)', None) == timeout,
-            callback=callback
-        )
+
+        firmware_version = self.info.get("firmware_version")
```

Comment:
Two things changed since this PR was created.
1. The field in info holding the firmware version was called `firmware_version`, now it is just `version`;
2. The field above does not contain the initial `v`;
Would you mind updating?